drivers/wireless/ieee80211/bcm43xxx/Kconfig

```diff
@@ -101,4 +101,12 @@ config IEEE80211_BROADCOM_NINTERFACES
 	default 1
 	depends on EXPERIMENTAL
 
+config IEEE80211_BROADCOM_SDIO_DMA_BUF_ALIGNMENT
```

Comment:
how about change to IEEE80211_BROADCOM_DMABUF_ALIGNMENT?

Reply:
Good. I've renamed it.